versioned_docs/version-v2.2.3/developers/components/atoms/ClusterHelper.jsx

```
@@ -0,0 +1,39 @@
+      Preload Cluster Node -
+      {Array.from({ length: 5 }, (_, index) => (
+        <button
+          style={{
+            background: selectedNode == index + 1 && 'blue',
+            color: selectedNode == index + 1 && 'white'
+          }}
+          onClick={() => {
+            setSelectedNode(index + 1)
+            setEndpointsValueUsingIndex(index + 1)
+          }}
+        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)